analytics/policy/services/registration_number_policies.py

```diff
@@ -0,0 +1,39 @@
+import re
+import json
+import urllib3
```

Comment:
No need to make request with urllib3 in the policies, since this is handled in API client

Reply:
So, it means that I need to remove the second method which is valid_registration_number.

Reply:
no, it means you can use the BusinessLicenceClient object to retrieve whatever you want to check with the given `registration_number`